core/model/src/main/java/org/activityinfo/model/expr/functions/DivideFunction.java

```diff
@@ -11,6 +11,12 @@ private DivideFunction() {
     }
 
     @Override
+    protected double apply(double a) {
+        // cannot apply to unary function
+        return Double.NaN;
```

Comment:
I don't think it's actually possible for this code to get called as the parser will not accept `*3` or `/3` but `NaN` doesn't seem to be the right result. You can just throw a new IllegalStateException() to indicate that something has gone really really wrong.